ui/packages/consul-ui/app/components/consul/exposed-path/list/index.hbs

```diff
@@ -37,7 +37,7 @@
         <dl class="port">
           <dt>
             <Tooltip>
-                Port
+              ListenerPort
```

Comment:
Why are changing this to a camel case tooltip?

Reply:
I wanted it to be clear that the port here is the `ListenerPort` not the `LocalPathPort`, using just `Port` could have meant either. Usually its best to use the exact same terms as the config so it's easy to match things up between the config and the UI. As a user I can see now that in order to change the value I see in the UI, I need to change the `ListenerPort` configuration value, previously I wouldn't have been sure whether I needed to change `ListenerPort`, `LocalPathPort` (or maybe even try using a non-existent `Port`)

Reply:
yes, but none of our tooltips are camel cased. This should be ` Listener Port`.

Reply:
Ah I getcha, k I added spaces to `ListenerPort` and `LocalPathPort` to make `Listener Port` and `Local Path Port`